platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -41,6 +41,9 @@ NS_ASSUME_NONNULL_BEGIN
  method returns `YES`, this camera becomes the map view’s camera.
  @return A Boolean value indicating whether the map view should stay at
  `oldCamera` or change to `newCamera`.
+ 
+ #### Related examples
+ See the <a href="https://www.mapbox.com/ios-sdk/maps/examples/constraining-gestures/">Restrict map panning to an area</a> example to learn how to use this method and `MGLMapCamera` objects to restrict a users ability to pan your map.
```

Comment:
This (and some of the next few in `MGLMapViewDelegate.h` didn’t get the wrapping treatment. 🙇

Reply:
😳😅